independent-publisher-connectors/OMDb API/apiDefinition.swagger.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "The OMDb API is a RESTful web service to obtain movie information, all content and images on the site are contributed and maintained by our users.\nThis API requires authorization, you can get a free key here: [http://omdbapi.com/apikey.aspx](http://omdbapi.com/apikey.aspx)",
+    "version": "1.0",
+    "title": "OMDb",
+    "termsOfService": "http://omdbapi.com/legal.htm",
+    "contact": {
+      "name": "Aaryan Arora",
+      "email": "aaryan2134@gmail.com"
+    }
+  },
+  "host": "omdbapi.com",
+  "basePath": "/",
+  "schemes": ["http", "https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/": {
+      "get": {
+        "tags": ["Search Parameter"],
+        "summary": "Returns an array of results for a given title or ID or Keyword",
+        "operationId": "GetSearchResults",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "apikey",
+            "required": true,
+            "type": "string",
+            "description": "API Access Key",
```

Comment:
all Parameters descriptions should end with a period. No period required only if there's a single word.

Reply:
Done